osh/hub/scan/fixtures/initial_data.json

```
@@ -270,6 +270,14 @@
     "name": "fedora-40-x86_64"
   }
 },
+{
+  "pk": 7,
+  "model": "scan.mockconfig",
+  "fields": {
+    "enabled": true,
```

On a second thought, I'll disable this feature by default because it is quite experimental.
```suggestion
    "enabled": false,
```

---

I would not worry too much about it because OSH instances with a fresh database are currently used only for experiments anyway.  If you disable it in the fixture, you might need to re-enable it in the CI scripts to keep the CI working.

---

Alright then! 😄